.changeset/green-suns-drive.md

```diff
@@ -0,0 +1,53 @@
+---
+"@astrojs/starlight": minor
+---
+
+Adds support for Astro v5, drops support for Astro v4.
+
+⚠️ **BREAKING CHANGE:** Astro v4 is no longer supported. Make sure you [update Astro](https://docs.astro.build/en/guides/upgrade-to/v5/) and any other official integrations at the same time as updating Starlight:
```

Comment:
```suggestion
## Upgrade Astro and dependencies

⚠️ **BREAKING CHANGE:** Astro v4 is no longer supported. Make sure you [update Astro](https://docs.astro.build/en/guides/upgrade-to/v5/) and any other official integrations at the same time as updating Starlight:
```
Not sure whether the headers are strictly speaking necessary, but it feels like this could be broken into 2 sections to make each part of the process obvious.

Reply:
Ah, I think headings might be tricky because once this ends up inside the CHANGELOG there’s an expected heading hierarchy (and this entire changeset will be inside a bullet list). We’d need to double check what level these have to be if we think a heading is important.